src/web/cursorUtils.ts

```diff
@@ -145,7 +145,19 @@ function scrollCursorIntoView(target: HTMLInputElement) {
     return;
   }
 
-  const caretRect = selection.getRangeAt(0).getClientRects()[0];
+  const caretRects = selection.getRangeAt(0).getClientRects();
+
+  // we'll find the caretRect from the DOMRectList above with the largest bottom value
+  let caretRect = caretRects[0];
+  if (caretRect) {
+    for (let i = 1; i < caretRects.length; i++) {
+      const ithCaretRect = caretRects[i];
+      if (ithCaretRect && ithCaretRect.bottom > caretRect.bottom) {
+        caretRect = ithCaretRect;
+      }
+    }
+  }
```

Comment:
How about changing variable names to more standardized ones, fe:

```suggestion
  let currentCaretRect = caretRects[0];
  if (currentCaretRect) {
    for (let i = 1; i < caretRects.length; i++) {
      const caretRect = caretRects[i];
      if (caretRect && caretRect.bottom > currentCaretRect.bottom) {
        currentCaretRect = caretRect;
      }
    }
  }
```